src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/MetricsFormattingTests.cs

```diff
@@ -44,14 +44,16 @@ public async Task HistogramFormat_Test()
         {
             List<ICounterPayload> payload = new();
 
-            payload.Add(new AggregatePercentilePayload(MeterName, InstrumentName, "DisplayName", string.Empty, string.Empty,
+            payload.Add(new AggregatePercentilePayload(new CounterMetadata(MeterName, InstrumentName, null, null, null), "DisplayName", string.Empty, string.Empty,
```

Comment:
nit: The latest version of the changes when they got merged in added labels for the nulls - maybe you pulled this from an older version?